### DISCUSSION TITLE
An overhaul of Discord Rules

### DISCUSSION THREAD

**Discussion by felix920506** [jellyfin/jellyfin-meta#69] [General]
The current set of Discord rules haven't been revised for over a year, and contain many shortcomings.
Non exhaustive list of shortcomings:

There is no way to contact moderators privately if one has the need to do so:
In cases of personal attacks and/or harassment, the act of contacting the mods will often make the situation worse if seen by the perpetrator.

There are no rules against the spread of potentially harmful technical information:
Discord is a special place of the community where there are many people that are relatively tech illiterate. Wrong or potentially harmful technical information can cause significantly more damage than they would otherwise cause on somewhere like Github.

EDIT: Determination on misinformation / potentially harmful information
There have been concerns brought up about how we would determine if a piece of information is harmful / wrong or not.
I think we can mostly trust the community to self moderate, and only take action for repeated demonstrations of unwillingness to take feedback. To further elaborate, consider the following cases.

Case 1 - Clear cut cases:
The community will immediately jump on the info and make a correction. There is no ambiguity in this case and the person should take the criticism and move on

Case 2 - Controversial cases:
The community will quickly spring into a debate providing evidence both ways. In this situation we shall do nothing more than simply direct the conversation to offtopic and keep it civilized. We shall only take mod action in case someone violates other rules during the debate.
(End of edit)

Below is the set of new rules I propose, based on the old rules. I am open to constructive feedback.

> Basics
> 
> 1. Treat everyone with respect. Absolutely no harassment, doxxing, witch hunting, sexism, racism, or hate speech allowed. You are only allowed to share information about others IF they have given you EXPLICIT and PRIOR permission to do so. (Edited)
> 2. This chat lives primarily on Matrix, and is bridged to Discord and IRC. You will see many people, including most of our team members, with "BOT" next to their names. These are humans, not bots. Our actual bot is called "Jeff (Bot)".
> 3. In rare cases, your first message to the chat may be dropped by the Matrix bridge. Please send a general "Hello" message in #offtopic before posting your questions to ⁠troubleshooting, or else we may miss it.
> 4. Do NOT DM team members or fellow users or send them friend requests unless you have been asked to do so. Please ask all questions in the wider community channels instead. If you wish to contact the team privately, please refer to the Contact and Punishments section
> 5. No spam or self-promotion (server invites, advertisements, etc) without permission from a staff member.
> 6. No discussion of media acquisition or related tools is allowed, regardless of its legality. Where your media comes from or how to get it is none of our or our community's business. (Edited)
> 7. No NSFW content of any medium allowed. This is an SFW and all-ages community.
> 8. There are cases where the rules don't cover. Please act in good faith under these cases. Moderation will be done at the team's discretion. (Edited)
> 
> 
> Technical Misinformation Rules
> 1. Definition of Technical Misinformation: Information about technology that can be objectively proven wrong.
> 2. You will generally not face punishment for simply saying something wrong occasionally. This rule is reserved for users who repeatedly demonstrate their unwillingness / inability to do basic research and / or take corrections from the community.
> 3. You may not spread information that may violate the ToS (Terms of services) of other services. When the ToS is unclear, you should assume that it is against the ToS. (Edited)
> 4. (Removed because unenforceable)
> 5. Discussion of related topics should be limited to #offtopic only.
> 
> 
> Contact and Punishments
> 1. If you see something against the rules, or something that makes you feel unsafe, let our staff know. We want this server to be a welcoming space for all users of Jellyfin.
> 2. If you wish to contact moderators privately, please send an E-mail to [team@jellyfin.org](mailto:team@jellyfin.org) containing all the relevant info on your matter.
> 3. Violation of these rules may result in the following punishments: an informal warning, a formal warning or a ban. If a formal warning has been issued, the next violation will result in a ban.
>
> The above rules are in guidance of our community guidelines. See https://jellyfin.org/docs/general/community-standards for more info. (Edited)

**Comment by JPVenson** [comment]
i would keep the 6th point very close to our official guidelines, explaining also talks about related tools and such are not allowed

**Reply by felix920506** [reply]
> No discussion of media acquisition or related tools is allowed, regardless of its legality. Where your media comes from or how to get it is none of our or our community's business.

Do you think this is a good amendment to this item?

**Reply by JPVenson** [reply]
works for me

**Comment by joshuaboniface** [comment]
Seems fine to me.

**Reply by felix920506** [reply]
Thanks! Ideally we should let this sit for a few days to collect more feedback. Once all is good we can remove the current set of rules on Discord and replace them with the new rules.

**Reply by felix920506** [reply]
I'm thinking listen for feedback until the end of the week and set them in effect some time next week. I will format it in discord markdown after the text is finalized.

**Comment by sevenrats** [comment]
I dont think there is any need to carve out this much space for violating the ToS of other projects.  Frankly the constant "abuse cloudflare" advice has been a blemish on our chat spaces for a long time and enshrining this behavior in policy is a bad idea. it should be simply against the rules to advise people to violate the ToS of other platforms.

**Reply by felix920506** [reply]
Edited.

**Comment by felix920506** [comment]
I was asked to try to reuse text from the forum rules.

> # Jellyfin Discord Server Rules
> Version 2.0
> ## Basics
> 1. You **shall** abide by our [Community Standards](https://jellyfin.org/docs/general/community-standards) at all times.
> 2. You **shall not** do any of the following things:
>   - Antagonize, flame, insult, demean, abuse, or harass other users or those outside of the community.
>   - Dox or otherwise share others' private information, even if it is available publicly.
>   - Post offensive, sexual, or otherwise inappropriate content - keep it "Safe For Work" in all channels.
>   - Discuss media acquisition or related tools, regardless of its legality. Where your media comes from or how to get it is none of our or our community's business.
>   - Spam the chat rooms in any way, including but not limited to Telegram short links and Self advertisement.
>   - Post verbatim replies from "AI" Chat systems (e.g. ChatGPT) as answers.
> 3. Your username and profile fields **shall** be appropriate with respect to our above standards and **shall not** contain swearing, slurs, piracy, or attempts to impersonate others.
> 4. You **shall** ensure you are posting in the correct channel before making a post. Do **NOT** post the same question in multiple channels.
> 5. Please write in English if at all possible, as English is the working language of the Jellyfin project. If you cannot, please write in your native language (with or without attempted English) and ask for a translation if required.
> 6. If you are using Jellyfin for commercial purposes or within a business setting, please review the Commercial Support Policy posted in the [forum rules](https://forum.jellyfin.org/t-jellyfin-forum-rules).
> 7. Considering that Discord is more likely to see users less experienced with tech, there are additional rules about technical misinformation. Please see the Technical Misinformation Rules below.
> 8. There are cases where the rules don't cover. Please act in good faith under all circumstances. Moderation will be done at the team's discretion when not covered by the rules.
> 
> ## Discord Specifics
> 1. This chat lives primarily on Matrix, and is bridged to Discord and IRC. You will see many people, including most of our team members, with "BOT" next to their names. These are humans, not bots. Our actual bot is called "Jeff (Bot)".
> 2. In rare cases, your first message to the chat may be dropped by the Matrix bridge. Please send a general "Hello" message in #offtopic before posting your questions to ⁠#troubleshooting, or else we may miss it.
> 3. Do NOT directly message team members or fellow users or send them friend requests unless you have been asked to do so. Please ask all questions in the wider community channels instead. If you wish to contact the team privately, please refer to the Contact and Punishments section
> 
> ## Technical Misinformation Rules
> 1. Definition of Technical Misinformation: Information about technology that can be objectively proven wrong.
> 2. You will generally **NOT** face punishment for simply saying something wrong occasionally. This rule is reserved for users who repeatedly demonstrate their unwillingness / inability to do basic research and / or take corrections from the community.
> 3. You may **NOT** spread information that may violate the ToS (Terms of services) of other services (eg. Hosting JF behind CloudFlare). When the ToS is unclear, you should assume that it is against the ToS.
> 4. Discussion of related topics should be limited to #offtopic only.
> 
> ## Contact and Punishments
> 1. If you see something against the rules, or something that makes you feel unsafe, let our staff know. We want this server to be a welcoming space for all users of Jellyfin.
> 2. If you wish to contact moderators privately, please send an E-mail to team@jellyfin.org containing all the relevant info on your matter.
> 3. Violation of these rules may result in the following punishments: an informal warning, a formal warning or a ban. If a formal warning has been issued, the next violation will result in a ban.

**Reply by joshuaboniface** [reply]
Sounds good to me!

**Reply by oddstr13** [reply]
discord specifics point 3, please expand DM to direct(ly) message

**Reply by felix920506** [reply]
@oddstr13 fixed

**Reply by thornbill** [reply]
Technical Misinformation Rules 4: what related topics?

**Reply by felix920506** [reply]
@thornbill fighting out what's right / wrong or what's allowed / not allowed in some ToS of another platform
